pkg/kubectl/cmd/apiversions.go

```diff
@@ -33,15 +32,15 @@ func NewCmdApiVersions(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"apiversions"},
 		Short:   "Print available API versions.",
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunApiVersions(f, out)
+			err := RunApiVersions(f, out, cmd)
 			cmdutil.CheckErr(err)
 		},
 	}
 	return cmd
 }
 
-func RunApiVersions(f *cmdutil.Factory, out io.Writer) error {
-	if os.Args[1] == "apiversions" {
+func RunApiVersions(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
+	if calledByAlias(cmd) {
```

Comment:
+1 on all aliases deprecated. I think we should go the git route on this, where in the future we can support custom aliases in your .kubectlrc, but officially it's one unaliased command per command to keep kubectl simple and to keep documentation, tutorials, blog posts etc. consistent.